src/Emulator/Device/deviceColors.ts

```diff
@@ -7,6 +7,8 @@ export enum DeviceColor {
   HOST = '#9E1059',
 }
 
+// type DeviceColorClass = 'Router' | 'r' | 'Switch' | 's' | 'Host' | 'h' |
```

Comment:
what is the intuition by having both `Router`and `r`?

Reply:
You're right. I actually could have used a previously defined type `DeviceType`. I made the necessary changes and I think that looks better!